GitHub Pull Request Demo (#2)
Robodox-599/2024_Lepidopterist: README.md

# Robodox 599 

## Overview
The Robodox 599 Robot is a Timed Robot with the following subsystems: 
- Drive: processing controller commands into motor movement

> - Intake: 
> - Indexer: a convyer belt like mechanism to move objects to the shooter

Author: ```suggestion
- Intake: wristed rollers that help pick up game pieces
- Indexer: a conveyer belt like mechanism to move game pieces to the shooter
```

- Shooter: angling the mechanism to shoot objects to fullfill mission criteria

> - Vision: cameras that help read [AprilTags](https://april.eecs.umich.edu/software/apriltag) and..

Author: ```suggestion
- Vision: cameras that help read [AprilTags](https://april.eecs.umich.edu/software/apriltag) 
```

In addition to robot code the team utulizes the following software tools: 
- [Advantage Kit](https://docs.advantagekit.org/) for robot testing and drive practice via simulation

## Getting Started
1. Install [WPILib](https://wpilib.org/) based on your OS
2. Fork and clone relevant [599 GitHub](https://github.com/Robodox-599) repo, look for the year you'd like to contribute to
3. (Opt) Download GitHub Desktop, used for extra git source control support (you can also use VS code)
4. To confirm cloning has been successful attempt building with ```./gradlew build```

## Contributing
- Fork the repoistory
- Create a branch based on the subsystem you are contributing to
- Modify code, commit, and push

> - Create a pull request

Author: ```suggestion
- Create a pull request

## 599 Programming Team
<!-- Add your name below this comment -->
```
